docs/conf.py

```diff
@@ -170,7 +170,7 @@
 
 # autodoc options
 autoclass_content = "class"  # only include docstring from Class (not __init__ method)
-autodoc_inherit_docstrings = False
+autodoc_inherit_docstrings = True
```

Comment:
Previously, the `@overrides` decorator copied a docstring to the overriden method if it didn't already have its own. We still want to see the docstring in the built sphinx docs, and this updating this configs seems to achieve that.